homework_03/05_Viktoriya_Vasileva/Circles.py

```diff
@@ -0,0 +1,69 @@
+import enum
+import math
+
+class RelativePosition(enum.Enum):
+    NO_COMMON_POINTS = 0
+    TOUCHING = 1
+    INTERSECTING = 2
+    #TOUCHING_INTERNALLY = 3
+    SAME = 3
+
+class Point:
+    def __init__(self, x, y):
+        assert isinstance(x, float) 
+        assert isinstance(y, float)
+        self.x = x
+        self.y = y
+
+    def line(self, point):
```

Comment:
I think that the idea to have a method for finding the distance between two points inside the Point class is great, so you receive my complaints here!